src/components/side-panel/layer-panel/layer-configurator.js

```diff
@@ -773,6 +769,53 @@ export default function LayerConfiguratorFactory(SourceDataSelector) {
       );
     }
 
+    _renderS2LayerConfig({
+      layer,
+      visConfiguratorProps,
+      layerConfiguratorProps,
+      layerChannelConfigProps
+    }) {
+      return (
+        <StyledLayerVisualConfigurator>
+          {/* Color */}
+          <LayerConfigGroup
+            {...layer.visConfigSettings.filled}
+            {...visConfiguratorProps}
+            label="Fill Color"
+            collapsible
+          >
+            {layer.config.colorField ? (
+              <LayerColorRangeSelector {...visConfiguratorProps} />
+            ) : (
+              <LayerColorSelector {...layerConfiguratorProps} />
+            )}
+            <ConfigGroupCollapsibleContent>
+              <ChannelByValueSelector
+                channel={layer.visualChannels.color}
+                {...layerChannelConfigProps}
+              />
+              <VisConfigSlider {...LAYER_VIS_CONFIGS.opacity} {...visConfiguratorProps} />
+            </ConfigGroupCollapsibleContent>
+          </LayerConfigGroup>
+
+          {/* height */}
+          <LayerConfigGroup {...LAYER_VIS_CONFIGS.enable3d} {...visConfiguratorProps} collapsible>
```

Comment:
to match h3 layer configuration, we should allow both `elevationScale` and `elevationRange`

```js
          <LayerConfigGroup {...LAYER_VIS_CONFIGS.enable3d} {...visConfiguratorProps} collapsible>
            <ChannelByValueSelector
              channel={layer.visualChannels.size}
              {...layerChannelConfigProps}
            />
            <ConfigGroupCollapsibleContent>
              <VisConfigSlider
                {...LAYER_VIS_CONFIGS.elevationScale}
                {...visConfiguratorProps}
                labl="Zoom scale"
              />
              <VisConfigSlider
                {...LAYER_VIS_CONFIGS.elevationRange}
                {...visConfiguratorProps}
                label="Height Range"
              />
            </ConfigGroupCollapsibleContent>
          </LayerConfigGroup>
```

Reply:
<img width="1111" alt="Screen Shot 2020-03-04 at 10 15 26 AM" src="https://user-images.githubusercontent.com/2554552/75909524-1c3d9e80-5e01-11ea-949d-48b269d932a9.png">

Reply:
Can you make lable of these to be consistent? e.g.
- Height based on
- Height Scale
- Elevation Scale
- Height Range

Reply:
<img width="1792" alt="Screen Shot 2020-03-12 at 12 25 39 PM" src="https://user-images.githubusercontent.com/2554552/76559620-b5e9f900-645c-11ea-9c2f-d28b60d255ab.png">